code/src/App.js

```diff
@@ -1,9 +1,89 @@
-import React from 'react'
+import React, { useState } from "react";
+import "./app.css"
+// import { Agequest } from "./QuestRadio";
+// import { Selectanimal } from "./QuestAnimal";
+// import { Myname } from "./QuestName";
+import { Summery } from "./Summery"
+import { Button } from "./Button"
+
+const ageGroups = ["18-25", "19-34", "35+"];
 
 export const App = () => {
+
+  const [name, setName] = useState("");
+  const [ageGroup, setAgeGroup] = useState();
+  const [animal, setAnimal] = useState('');
```

Comment:
You are using both `"` and `'`, it would be awesome if you choose one and stick to it